drivers/ad9144/ad9144.c

```diff
@@ -492,6 +492,14 @@ int32_t ad9144_setup(struct ad9144_dev **device,
 	ad9144_spi_write(dev, REG_SYNC_CTRL, 0x01);	// sync-oneshot mode
 	ad9144_spi_write(dev, REG_SYNC_CTRL, 0x81);	// sync-enable
 	ad9144_spi_write(dev, REG_SYNC_CTRL, 0xc1);	// sysref-armed
+	ad9144_spi_write(dev, REG_XBAR_LN_0_1,
+			init_param->jesd204_lane_xbar[0] | init_param->jesd204_lane_xbar[1]<<3);
```

Comment:
Please use the SRC_LANE0(), SRC_LANE1() MACROs instead of direct shifting. Do it similar for the other lanes.